### PR TITLE
Update image.schema.json

### PR DESCRIPTION
**Pull request by UlrikeS91**
- added `voxelSize` and `unit`

Questions: 
1) Does the description for `voxelSize` makes sense?
2) Should they be required or optional?

### CODE REVIEW

**Comment by UlrikeS91** [comment]
related to #79

**Comment by UlrikeS91** [comment]
Sorry about the slight chaos...This is what happens when you temporarily forget how the alphabet works -.-

**Review by skoehnen — APPROVED** [review]
Looks good to me

File: v0.1/elements/image.schema.json [hunk 1]
```diff
@@ -31,9 +33,18 @@
...
-      "definition": "Indexing parameters that apply to the image.",
-      "$ref": "https://raw.githubusercontent.com/HumanBrainProject/SANDS/master/v0.1/definitions.json#definitions/INDEXING_PARAMETERS"
-    }	  
+    "unit": {
+      "description": "A determinate quantity adopted as a standard of the measurement.",
+      "$ref": "https://raw.githubusercontent.com/HumanBrainProject/SANDS/master/v0.1/definitions.json#definitions/CONTROLLED_TERM"
+    },    
+    "voxelSize": {
+      "type": "array",
+      "description": "Size of one voxel (3D) or pixel (2D, with z = 0).",
```
Comment (skoehnen) [inline v0.1/elements/image.schema.json:42]: Maybe we should mention that the voxelSize is in relation to the unit.
For example: "Size of one voxel (3D) or pixel (2D, with z = 0), in the unit defined for this image"

Comment (UlrikeS91) [reply v0.1/elements/image.schema.json:42]: I like that suggestion. I'll quickly add it :)

Comment (skoehnen) [reply v0.1/elements/image.schema.json]: We could also specify that it is the size of the voxel in the physical space, that would be as specific as possible.

Comment (skoehnen) [reply v0.1/elements/image.schema.json]: > Corresponding size of one voxel (3D) or pixel (2D, with z = 0) in the physical space, defined in the unit of this image.

**Comment by UlrikeS91** [comment]
@skoehnen Added your suggestion to the description. We can merge it into the master, right?

**Comment by skoehnen** [comment]
> Questions:
> 
> 1. Does the description for `voxelSize` makes sense?
I added an alternative version @UlrikeS91 do you like this better
> 2. Should they be required or optional?
I think required would be the best way, but we can think about putting it in optional.

File: v0.1/elements/image.schema.json (hunk 1 above)
Comment (UlrikeS91) [reply v0.1/elements/image.schema.json]: Even better! Changed the description.

**Review by skoehnen — APPROVED** [review]

**Pull request merged**